src/bin/tikv-server.rs

```diff
@@ -79,21 +79,27 @@ use tikv::util::rocksdb::metrics_flusher::{MetricsFlusher, DEFAULT_FLUSER_INTERV
 
 const RESERVED_OPEN_FDS: u64 = 1000;
 
-fn exit_with_err<E: Error>(e: E) -> ! {
-    exit_with_msg(format!("{:?}", e))
-}
-
-fn exit_with_msg(msg: String) -> ! {
-    error!("{}", msg);
-    process::exit(1)
+macro_rules! fatal {
```

Comment:
I have no objection to the macro, since it is not exported. How about you @siddontang ?